.github/workflows/ci.yml

```
@@ -0,0 +1,81 @@
+          DEBUG: True
+          DB_NAME: test_db
+          DB_USER: test_user
+          DB_PASSWORD: test_password
+          DB_HOST: localhost # The service containers are available on localhost within the job
+          DB_PORT: 5432
+          CELERY_BROKER_URL: redis://localhost:6379/0
+          CELERY_RESULT_BACKEND: redis://localhost:6379/0
+
+      - name: Login to Docker Hub
+        if: success() # Only run if tests pass
+        uses: docker/login-action@v2
```

actionlint (1.7.9)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)